source/imgui_impl_opengl3.h

```diff
@@ -45,7 +45,7 @@ IMGUI_IMPL_API void     ImGui_ImplOpenGL3_DestroyDeviceObjects();
 // If auto-detection fails or doesn't select the same GL loader file as used by your application,
 // you are likely to get a crash in ImGui_ImplOpenGL3_Init().
 // You can explicitly select a loader by using one of the '#define IMGUI_IMPL_OPENGL_LOADER_XXX' in imconfig.h or compiler command-line.
-#define IMGUI_IMPL_OPENGL_LOADER_GLEW
+#define IMGUI_IMPL_OPENGL_ES2
```

Comment:
This can be easily changed into "auto-detect".

Delete the whole #if thingy until the end of the file and replace it with

```
#if defined(__EMSCRIPTEN__)
#define IMGUI_IMPL_OPENGL_ES2
#else
#define IMGUI_IMPL_OPENGL_LOADER_GLEW
#endif
```